dhcpv4/option_userclass.go

```diff
@@ -66,6 +66,17 @@ func ParseOptUserClass(data []byte) (*OptUserClass, error) {
 			totalLength, len(data))
 	}
 
+	// Check if option is Microsoft style instead of RFC compliant
```

Comment:
can you please add a reference to the issue? Either a reference to https://github.com/insomniacslk/dhcp/issues/113 or to the links you posted there. Thanks!

Reply:
also mention the heuristic (similarly to how it's done in dnsmasq)